docs/UserGuide.md

```diff
@@ -119,15 +119,15 @@ Name: John Street
 Expected outcome for CLI:
 ```
 Name: John Doe (User)
```

Comment:
Are we going to go with John Doe and friends? Or use the sample names?

Reply:
We can always change it later after discussion.